site/gulpfile.js

```diff
@@ -1,15 +1,18 @@
 var gulp = require('gulp');
 var elixir = require('laravel-elixir');
+var argv = require('yargs').argv;
```

Comment:
We should probably add this as an explicit `package.json` dependency as well even if it happens to be pulled in by Elixir.